templates/bot.mustache

```diff
@@ -9,8 +9,14 @@
     {{> bot_css }}
 </head>
 
+<script>
```

Comment:
are you sure that we exactly need to use .mustache file? i think you can check the codebase and how we work with the same things, probably you can find better place to write it

Reply:
So `src` for images hardcoded in mustache template
If we have any way to resolve it or some examples, please share where I can find it @maryia-matskevich-deriv 🙏 
Unfortunately I didn't found any other cases

I only noticed that we use build strategy to build path for src, but we need a `window.location` information in web page to resolve this logic.